src/main/resources/templates/repo/agent/bedrock_agent_template.json

```diff
@@ -141,6 +141,19 @@
+				"PromptOverrideConfiguration": {
+					"PromptConfigurations": [
+						{
+							"BasePromptTemplate":"    {\n        \"anthropic_version\": \"bedrock-2023-05-31\",\n        \"system\": \"\n$instruction$\nYou have been provided with a set of functions to answer the user's question.\nYou must call the functions in the format below:\n<function_calls>\n  <invoke>\n    <tool_name>$TOOL_NAME</tool_name>\n    <parameters>\n      <$PARAMETER_NAME>$PARAMETER_VALUE</$PARAMETER_NAME>\n      ...\n    </parameters>\n  </invoke>\n</function_calls>\nHere are the functions available:\n<functions>\n  $tools$\n</functions>\nYou will ALWAYS follow the below guidelines when you are answering a question:\n<guidelines>\n- Think through the user's question, extract all data from the question and the previous conversations before creating a plan.\n- ALWAYS optimize the plan by using multiple functions <invoke> at the same time whenever possible.\n- Never assume any parameter values while invoking a function. Only use parameter values that are provided by the user or a given instruction (such as knowledge base or code interpreter).\n$ask_user_missing_information$\n- Always refer to the function calling schema when asking followup questions. Prefer to ask for all the missing information at once.\n- Provide your final answer to the user's question within <answer></answer> xml tags.\n$action_kb_guideline$\n$knowledge_base_guideline$\n$code_interpreter_guideline$\n$output_format_guideline$\n</guidelines>\n$knowledge_base_additional_guideline$\n$code_interpreter_files$\n$memory_guideline$\n$memory_content$\n$memory_action_guideline$\n$prompt_session_attributes$\n\",\n        \"messages\": [\n            {\n                \"role\" : \"user\",\n                \"content\" : \"$question$\"\n            },\n            {\n                \"role\" : \"assistant\",\n                \"content\" : \"$agent_scratchpad$\"\n            }\n        ]\n    }",
+							"InferenceConfiguration": {
+								"Temperature": 0
+							},
+							"ParserMode": "DEFAULT",
+							"PromptCreationMode": "OVERRIDDEN",
+							"PromptType" : "ORCHESTRATION"
+						}
+					]
+				},
```

Comment:
Please double check the template correctness from the trace of the agent. Also please be more specific on what the agent can disclose to the user (e.g. the agent can give a brief description of the tools available but the general system prompt for avoiding disclosing internal prompts and instructions should still be there.)